variables.tf

```diff
@@ -620,6 +620,26 @@ variable "karpenter" {
   description = "Allows to create/deploy/configure karpenter operator and its resources to have custom node auto-calling"
 }
 
+variable "keda" {
+  type = object({
+    enabled          = optional(bool, false)
+    name             = optional(string, "keda")   # keda chart name,
+    namespace        = optional(string, "keda")   # keda chart namespace
+    create_namespace = optional(bool, true)       # create keda chart
+    keda_version     = optional(string, "2.16.1") # chart version
+    scaling_type     = optional(string, "sqs")
```

Comment:
can we have instead of scaling_type some kind of enable/attach policies object which can have multiple options in future to add, the  sqs will be one of options, something like this:
```terraform
...
attach_policies = optional(object({
sqs = bool
}), {sqs= false})
...
```